docs/components/board/customlinux.md

```diff
@@ -9,11 +9,27 @@ tags: ["board", "components"]
 # SMEs: Susmita
```

Comment:
Not sure this is right: I think of myself as the SME with Olivia being a second choice and Susmita being my third choice.

Reply:
```suggestion
# SMEs: Alan, Olivia, Susmita
```